zdm/energetics.py

```diff
@@ -19,6 +21,35 @@ def init_igamma_splines(gammas, reinit=False):
             # iGamma
             igamma_splines[gamma] = interpolate.splrep(avals, numer)
 
+def init_igamma_linear(gammas, reinit=False, log=False):
+
```

Comment:
add quick doco line on log (yes, I know this was not originally documented, but we are slowly trying to remedy this!)

Reply:
done